dashboards-observability/public/components/event_analytics/explorer/events_views/doc_flyout.tsx

```diff
@@ -69,7 +74,15 @@ export const DocFlyout = ({
 
   const flyoutHeader = (
     <EuiFlyoutHeader hasBorder>
-      <EuiFlexGroup>
+      <EuiFlexGroup alignItems="center" gutterSize="xs">
+        {backButtonExists && (
```

Comment:
Does `backButtonExists` indicate that log patterns is shown in the flyout? If so let's change the variable name.